apps/core0/helper/socat/socat.go

```diff
@@ -75,19 +75,25 @@ type rule struct {
 	source source
 	port   int
 	ip     string
+	nic    string
 }
 
 func (r rule) Rule() string {
-	return fmt.Sprintf("%s dnat to %s:%d", r.source, r.ip, r.port)
+	return fmt.Sprintf("iif != \"%s\" %s dnat to %s:%d", r.nic, r.source, r.ip, r.port)
```

Comment:
r.source is now ignored? I beleive we should only do this incase r.source.ip == '0.0.0.0'

Reply:
nope, not ignored, it just added to the matching we already do!